File: api_service/models/forum.py
import orjson
from pydantic import BaseModel

from models.common import orjson_dumps


class Forum(BaseModel):
    slug: str
    title: str
    user: str
    posts: int = 0
    threads: int = 0

    class Config:
        # Заменяем стандартную работу с json на более быструю
        json_loads = orjson.loads
        json_dumps = orjson_dumps

File: api_service/models/forum.go
package models

//easyjson:json
type Forum struct {
	Slug    string `json:"slug,omitempty"`
	Title   string `json:"title,omitempty"`
	User    string `json:"user,omitempty"`
	Posts   int64  `json:"posts,omitempty"`
	Threads int64  `json:"threads,omitempty"`
}
